api/v1alpha1/customruntimeenvironment_types.go

```diff
@@ -40,11 +40,11 @@ const (
 	GitRepository BuildType = "GitRepository"
 )
 
-// CNBi Annotations is a list of annotations that are added to the custom notebook image
+// Open Data Hub Annotations is a list of annotations that are added to the custom runtime environment container image
```

Comment:
runtime environment container image is a bit redondant, isn't it ? Do we need `container` here ?